packages/bootstrap/scss/_variables.scss

```diff
@@ -1526,6 +1526,7 @@ $notification-error-border: tint($error, 9) !default;
 // Tooltip
 $tooltip-padding-y: $padding-y !default;
 $tooltip-padding-x: $padding-x !default;
+$tooltip-max-width: 250px !default;
```

Comment:
For the Bootstrap theme, maybe we should keep the tooltip max-width of the Twitter Bootstrap, which is 200px